<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数字图像处理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="fangda">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon/YY_fangda_1.png</normaloff>icon/YY_fangda_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="jingxiang">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon/镜像.png</normaloff>icon/镜像.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="suoxiao">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon/缩小.png</normaloff>icon/缩小.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="xuanzhuan">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icon/3.1-旋转.png</normaloff>icon/3.1-旋转.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>几何变换：</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>480</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>平滑处理：</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="image_show">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>700</width>
      <height>680</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>680</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>680</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>亮度调节：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="light_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>对比度调节：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="contrast_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>饱和度调节：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sat_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>121</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="recover">
       <property name="text">
        <string>恢复原图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="color_change_gray">
       <property name="text">
        <string>灰度化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="binary">
       <property name="text">
        <string>二值化</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>直方图处理：</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="rgb_hist">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>直方图均衡化</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>410</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>490</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>锐化处理：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>加性噪声：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guass_noise">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯噪声</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sp_noise">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>椒盐噪声</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="media_filter">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>330</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>中值滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mean_filter">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>均值滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gauss_filter">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>高斯滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="QRcode_pos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>二维码定位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Prewitt">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Prewitt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Canny">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>560</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/文件夹.png</normaloff>icon/文件夹.png</iconset>
    </property>
    <addaction name="open_file"/>
    <addaction name="separator"/>
    <addaction name="save_file"/>
   </widget>
   <widget class="QMenu" name="withdraw">
    <property name="title">
     <string>撤回</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/撤回.png</normaloff>icon/撤回.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="hist">
    <property name="title">
     <string>直方图</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/直方图.png</normaloff>icon/直方图.png</iconset>
    </property>
    <addaction name="gray_hist"/>
    <addaction name="separator"/>
    <addaction name="color_hist"/>
   </widget>
   <addaction name="file"/>
   <addaction name="withdraw"/>
   <addaction name="hist"/>
  </widget>
  <action name="open_file">
   <property name="icon">
    <iconset>
     <normaloff>icon/open.png</normaloff>icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="save_file">
   <property name="icon">
    <iconset>
     <normaloff>icon/保存.png</normaloff>icon/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="gray_hist">
   <property name="text">
    <string>灰度</string>
   </property>
  </action>
  <action name="color_hist">
   <property name="text">
    <string>彩色</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
